crates/rune/src/ast/attribute.rs

```diff
@@ -60,6 +60,13 @@ pub struct Attribute {
     /// The `]` character
     pub close: T![']'],
 }
+impl Attribute {
+    pub(crate) fn stream_span(&self) -> Span {
```

Comment:
Maybe this should be called `input_span`?

Reply:
maybe, the problem is, it'll end up in a MacroContext where it will be the "stream_span" maybe we should add a new AttributeMacroContext, but that would be a problem with the requirnment of passing macro cobtext around everywhere. AFAICT this is for error spans, but one could maybe also just return an error that says "Ihave no span yet give me the surrounding one" and the parrent function would apply the correct span instead of passing through the macrocontext to provide the span.

Reply:
I was just thinking that it reflects the name of the field. At the very least they should be named the same 😄.

Reply:
I mean one could argue that `input` is the correct name for function macros as well

Reply:
but not sure what you think of that breaking change.